Dockerfile.pipelines

```
@@ -93,10 +98,10 @@ RUN chown -R devito /home/devito
 USER devito
 WORKDIR /home/devito
 
-RUN if [ $installWithPip == "true" ] ; then \ 
-      python setup.py install --user ; \
+RUN if [ $installWithPip == "true" ] ; then \
+      python3 setup.py install --user ; \
       if [ $MPI_INSTALL == '1' ] ; then \
-        pip install --user -e .[extras] ; \
+        pip3 install --user -e .[extras] ; \
```

```suggestion
        python3 -m pip install --user -e .[extras] ; \
```

This seems to be the way the world is going..